app/controllers/series_controller.rb

```diff
@@ -16,21 +16,20 @@ def load_series
   # GET /series
   # GET /series.xml
   def index
-    if params[:user_id]
-      @user = User.find_by(login: params[:user_id])
-      unless @user
-        raise ActiveRecord::RecordNotFound, "Couldn't find user '#{params[:user_id]}'"
-      end
-      @page_subtitle = ts("%{username} - Series", username: @user.login)
-      pseuds = @user.pseuds
-      if params[:pseud_id]
-        @pseud = @user.pseuds.find_by(name: params[:pseud_id])
-        unless @pseud
-          raise ActiveRecord::RecordNotFound, "Couldn't find pseud '#{params[:pseud_id]}'"
-        end
-        @page_subtitle = ts("by ") + @pseud.byline
-        pseuds = [@pseud]
-      end
+    @user = User.find_by(login: params[:user_id]) if params[:user_id]
+    unless @user
+      flash[:error] = ts("Whose series did you want to see?")
+      redirect_to(root_path) and return
+    end
+
+    @page_subtitle = ts("%{username} - Series", username: @user.login)
+    pseuds = @user.pseuds
+    if params[:pseud_id]
+      @pseud = @user.pseuds.find_by(name: params[:pseud_id])
+      raise ActiveRecord::RecordNotFound, "Couldn't find pseud '#{params[:pseud_id]}'" unless @pseud
+      
+      @page_subtitle = ts("by ") + @pseud.byline
```

Comment:
This page_subtitle is only "by pseud (user)", it should probably include the word "series" somewhere, but that is outside the scope of this PR. Is the [support form](https://archiveofourown.org/support) the right place to report issues like this?

Reply:
Here is fine! I've made [an issue](https://otwarchive.atlassian.net/browse/AO3-6378).